lib/tasks/import.rake

```diff
@@ -0,0 +1,18 @@
+namespace :import do
+  desc 'Imports transactions from your Monzo account'
+  task :monzo => :environment do
+    # Configure these environment variables on your Heroku instance
+    monzo_access_token = ENV.fetch('MONZO_ACCESS_TOKEN')
```

Comment:
One of the problems with this is that they expire very quickly, so it basically defeats the point of having it in an ENV var.

Reply:
Hmm, if that's the case, you can still pass it in when you actually run the script I guess.

Reply:
I think that would be for the best with all the params. I think keeping the env vars separate for the most part is a good idea, as they are generally specifically designed for webhook integrations. People got very confused in the past when we used env vars for both.

Reply:
But i love the general idea of using rake commands for it. I think it will help a lot!

Reply:
We could look into something like https://github.com/commander-rb/commander too